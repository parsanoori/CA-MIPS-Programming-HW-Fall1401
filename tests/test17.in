6
669654820
755910501
123000763
1023592991
730260314
640709759