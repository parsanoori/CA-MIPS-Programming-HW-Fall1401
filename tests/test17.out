123000763
640709759
669654820
730260314
755910501
1023592991